src/pfs/pfs.proto

```diff
@@ -58,15 +58,13 @@ message RepoAuthInfo {
 }
 
 message BranchInfo {
+  reserved 1;
```

Comment:
I don't think we need to reserve this with 2.0 since there is no expectation of compatibility. I kind of think that we would benefit from renumbering all of our protos for 2.0, then being stricter about the reserved numbers moving forward.

Reply:
I'll leave that for a separate PR because we'd need to sweep all the proto files anyway.